apps/aechannel/src/aesc_fsm_min_depth_watcher.erl

```diff
@@ -104,9 +105,10 @@
                                   , locked_until := aesc_channels:locked_until() }.
 
 -type tx_log_entry_key() :: {tx_hash(), block_hash()}.
--type tx_log_entry_value() :: #{ tx_hash := tx_hash()
-                               , block_hash := block_hash()
-                               , block_origin := chain }.
+-type tx_log_entry_value() :: #{ tx           => aetx_sign:signed_tx()
```

Comment:
A note here: when we log a tx_event, we don't have the full SignedTx from the start, and we don't necessarily want to fetch it right away. So it's added on-demand when `get_latest_tx()` is called.

Reply:
Also, no need to store the `tx_hash` in the `value` part, since it's already part of the key. We do want the full tx, though, when it's time to report something.